annotation/annotation/errors.py

```diff
@@ -158,3 +163,10 @@ def agreement_score_service_error_handler(
         status_code=400,
         content={"detail": f"Agreement score service error. {exc}"},
     )
+
+
+def cyclic_parent_error_handler(request: Request, exc: CyclicParentError):
+    return JSONResponse(
+        status_code=400,
+        content={"detail": f"Cyclic parent error. {exc.message}"},
+    )
```

Comment:
Are we sure it is a case for a lot of places? I think only when we create and modify, probably the handler is an overkill